cms/jinja2/templates/pages/methodology_page.html

```diff
@@ -0,0 +1,131 @@
+                    </div>
+                {% endif %}
+            </div>
+        {% endblock %}
+    </div>
+{% endblock %}
+
+{% block main %}
+    <div class="ons-container">
+        <div class="ons-grid ons-grid-flex-gap ons-grid-flex-gap--32">
+            <div class="ons-grid__col ons-grid__col--sticky@m ons-col-4@m">
+                {% with toc_title=_("Contents"), toc_aria_label=_("Sections in this page") %}
+                    {# fmt:off #}
+                    {{-
+                        onsTableOfContents({
+                            "title": toc_title,
+                            "ariaLabel": toc_aria_label,
+                            "itemsList": table_of_contents
+                        })
+                    }}
+                    {# fmt:on #}
+                {% endwith %}
+            </div>
+
+            <div class="ons-grid__col ons-col-8@m">
+                {# TODO: is the below comment and code still relevant? Can't find the variable in that file #}
+                {# if there are no contact details on the page, we don't want the last
+                streamfield block to have a bottom margin. last_flush is used in stream_block.html #}
+                {% if page.contact_details %}
+                    {% set last_flush = False %}
+                {% else %}
+                    {% set last_flush = True %}
+                {% endif %}
+
+                {% include_block page.content %}
+
+                {% if page.related_pages %}
+                {# fmt:off #}
+                {{-
+                    onsRelatedContent({
+                        "ariaLabel": _('Related publications'),
+                        "rows": [{
+                            "id": 'related-content',
+                            "title": _('Related publications'),
+                            "itemsList": page.related_pages
+                        }]
+                    })
+                -}} 
+                {# fmt:on #}
+                {% endif %}
+
+                {% if page.show_cite_this_page %}
+                    <section id="cite-this-page">
+                        <h2 class="ons-u-fs-l">{{ _("Cite this methodology") }}</h2>
+                        <p>
+                            {%- set cite_link -%}
+                                <a href="{{ fullpageurl(page) }}">{{ page.title }}</a>
+                            {% endset %}
+                            {% set latest_date = page.last_revised_date if page.last_revised_date else page.published_date %}
+                            {% trans trimmed latest_date_formatted = latest_date|date("DATE_FORMAT"), cite_link=cite_link %}
+                                Office for National Statistics (ONS), last revised {{ latest_date_formatted }}, ONS website, methodology, {{ cite_link }}
+                            {% endtrans %}
+                        </p>
+                    </section>
+                {% endif %}
```

Comment:
suggestion: this could now move to an include, and update on the analysis page too.
For the include, I'd update the trans block as:

```jinja2
{% trans trimmed latest_date_formatted = latest_date|date("DATE_FORMAT"), cite_link=cite_link page_type=page_type %}
    Office for National Statistics (ONS), last revised {{ latest_date_formatted }}, ONS website, {{ page_type }}, {{ cite_link }}
{% endtrans %}
```

with `page_type` being passed on to the include. So something like:

```jinja2
{% if page.show_cite_this_page %}
    {% with page_type=_("methodology") %}
        {% include "templates/components/cite-this/cite_this_page.html" %}
    {% endwith %}
{% endif %}
```

same for the contact details include. Could be one for the front-end task and if so, best to add notes on the relevant ticket

Reply:
Had a chat about it with @zerolab  and decided not to go forward with this as the text for is different for Analysis and Methodology pages, so only a few words would be actually reused. 
(Analysis page has `released {{date}}` and Methodology page has `last revised {{date}}`)